api4/config.go

```diff
@@ -128,6 +126,46 @@ func updateConfig(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte(cfg.ToJson()))
 }
 
+func filterConfigByPermission(c *Context, structField reflect.StructField, parentStructFieldName string) bool {
```

Comment:
I believe you can remove `parentStructFieldName` now

Reply:
Done. Thanks.